network/p2p/gossip/handler.go

```diff
@@ -70,14 +99,15 @@ func (h Handler[T]) AppRequest(_ context.Context, _ ids.NodeID, _ time.Time, req
 			return false
 		}
 
-		// check that this doesn't exceed our maximum configured response size
+		// check that this doesn't exceed our maximum configured target response
+		// size
+		gossipBytes = append(gossipBytes, bytes)
+
 		responseSize += len(bytes)
-		if responseSize > h.maxResponseSize {
+		if responseSize > h.targetResponseSize {
 			return false
```

Comment:
are the targetResponseChanges meant to be included in this PR?

Reply:
from diff on https://github.com/ava-labs/avalanchego/pull/1995, should be gone now that it's merged